src/redux/auth/auth.sagas.js

```diff
@@ -50,13 +50,13 @@ function* handleAdminCheckByToken() {
   } catch (error) {
     yield put(setAuthLoading(false));
     yield put(setAuth(false));
-    clearLocalStorage();
+    yield call(clearLocalStorage,null);
```

Comment:
There is no need to use `yield call` for access to the local storage because it is sync action from the js point of view

Reply:
fixed